Content.Server/PneumaticCannon/PneumaticCannonSystem.cs

```diff
@@ -105,6 +108,11 @@ private void OnShoot(Entity<PneumaticCannonComponent> cannon, ref GunShotEvent a
         _slots.TryEject(uid, PneumaticCannonComponent.TankSlotId, args.User, out _);
     }
 
+    private void OnGunRefreshModifiers(Entity<PneumaticCannonComponent> ent, ref GunRefreshModifiersEvent args)
+    {
+
+    }
+
```

Comment:
Head empty?

Reply:
Forgot to add that part of the diff to the commit, fixed in [`0b6ded7` (#24077)](https://github.com/space-wizards/space-station-14/pull/24077/commits/0b6ded789c63ccc5b9a796fdd024feb35b5d6da8)